docs/INSTALL.md

```diff
@@ -0,0 +1,73 @@
+# 1. <a name='Pre-BuiltPackages'></a>Installation using Pre-Built Packages
+
+**OpenVINO™ integration with TensorFlow** has two releases: one built with CXX11_ABI=0 and another built with CXX11_ABI=1.
+
+Since TensorFlow packages available in [PyPi](https://pypi.org) are built with CXX11_ABI=0 and OpenVINO™ release packages are built with CXX11_ABI=1, binary releases of these packages **cannot be installed together**. Based on your needs, you can choose one of the two available methods:
+
+- **OpenVINO™ integration with TensorFlow** alongside PyPi TensorFlow
+  (CXX11_ABI=0, no OpenVINO™ installation required, no VAD-M support)
+
+- **OpenVINO™ integration with TensorFlow** alongside the Intel® Distribution of OpenVINO™ Toolkit
+  (CXX11_ABI=1, needs a custom TensorFlow package, enables VAD-M support)  
+<br/>  
+
+## <a name='Prebuiltpackagessummary'></a>Pre-built packages summary
+  
+|TensorFlow Pip Package| **OpenVINO™ integration with TensorFlow** Pip Package|Supported OpenVINO™ Flavor|Supported Hardware Backends|Comments|
+| -----------------|-----------------------------------|----------------------------|---------------------------|----------------|
+|tensorflow| openvino-tensorflow|OpenVINO™ built from source|CPU,GPU,MYRIAD|**OpenVINO™** libraries are built from source and included in the wheel package|
+|tensorflow-abi1| openvino-tensorflow-abi1|Dynamically links to OpenVINO™ binary release|CPU,GPU,MYRIAD,VAD-M|**OpenVINO™ integration with TensorFlow** libraries are dynamically linked to OpenVINO™ binaries|
+<br/>  
+
+##  1.1. <a name='InstallOpenVINOintegrationwithTensorFlowalongsidePyPiTensorFlow'></a>Install **OpenVINO™ integration with TensorFlow** alongside PyPi TensorFlow
+
+This **OpenVINO™ integration with TensorFlow** package includes pre-built libraries of OpenVINO™ version 2021.4. The users do not have to install OpenVINO™ separately. This package supports Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs, and Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs).
+
+
+        pip3 install pip==21.0.1
+        pip3 install tensorflow==2.5.0
+        pip3 install openvino-tensorflow
+<br/> 
+
+##  1.2. <a name='InstallOpenVINOintegrationwithTensorFlowalongsidetheIntelDistributionofOpenVINOToolkit'></a>Install **OpenVINO™ integration with TensorFlow** alongside the Intel® Distribution of OpenVINO™ Toolkit
+
+This **OpenVINO™ integration with TensorFlow** package is currently compatible with OpenVINO™ version 2021.4. This package supports Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs, Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs), and Intel<sup>®</sup> Vision Accelerator Design with Movidius™ (VAD-M).
+
+You can build [TensorFlow from source](https://github.com/openvinotoolkit/openvino_tensorflow/blob/master/docs/BUILD.md#tensorflow) with -D_GLIBCXX_USE_CXX11_ABI=1 or use the following TensorFlow package:
```

Comment:
Last part should be " or follow below instructions to install appropriate Tensorflow"